lib/main/bindings/network/binding_tag.dart

```diff
@@ -2,4 +2,5 @@ class BindingTag {
   static const isolateTag = 'isolateTag';
   static const cleanUpPublicAssetsInteractorBindingsTag = 'CleanUpPublicAssetsInteractorBindingsTag';
   static const publicAssetBindingsTag = 'PublicAssetBindingsTag';
+  static const restoreIdentityCacheInteractorBindingsTag = 'RestoreIdentityCacheInteractorBindingsTag';
```

Comment:
why we need this tag?

Reply:
`IdentitiesController` and `IdentityCreatorController` shares several bindings. If there is no tag, when `IdentityCreatorController` closes, some dependencies of `IdentitiesController` will be disposed.